package.json

```diff
@@ -4,9 +4,15 @@
 	"description": "",
 	"main": "src/index.js",
 	"scripts": {
-		"start": "node ."
+		"start": "node .",
+		"start:dev": "nodemon --watch src"
 	},
 	"author": "",
 	"license": "ISC",
-	"dependencies": {}
+	"dependencies": {
+		"express": "^4.18.2",
+		"nodemon": "^3.0.1",
+		"swagger-ui-express": "^5.0.0",
+		"yaml": "^2.3.2"
```

Comment:
Why do you need yaml package?

Reply:
nodemon package is intended for local development, so it should be in the DevDependencies section of the package.json.

Reply:
> Why do you need yaml package?

Forgot to delete, was trying to parse yaml file